src/install/npm.zig

```diff
@@ -846,8 +846,16 @@ pub const PackageManifest = struct {
                 const versions = versions_q.expr.data.e_object.properties.slice();
                 for (versions) |prop| {
                     const version_name = prop.key.?.asString(allocator) orelse continue;
+                    const sliced_version = SlicedString.init(version_name, version_name);
+                    const parsed_version = Semver.Version.parse(sliced_version, allocator);
```

Comment:
This leaks some memory. We should probably use an arena in a few places here

Reply:
Actually, that `allocator` in `parse()` is unused historical artifact &minus; updated PR to trim it.

(OTOH, `Semver.Query.parse()` does require memory allocation &minus; however its return type comes with `.deinit()` so we are covered there)